backend/internetarchive/internetarchive.go

```diff
@@ -151,6 +151,11 @@ Owner is able to add custom keys. Metadata feature grabs all the keys including
 			Help:     "Host of InternetArchive Frontend.\n\nLeave blank for default value.",
 			Default:  "https://archive.org",
 			Advanced: true,
+		}, {
+			Name:     "metadata",
+			Help:     "Metadata to be set on the item.\n\nFormat is key=value. Note: x-archive-meta- prefix is automatically added.",
+			Default:  []string{},
```

Comment:
I am not 100% sure `[]string` works properly in backend config. Try to set multiple metadatas at once.

I think you might have to use `fs.CommaSepList` here instead

Reply:
Multiple metadatas at once works file. I assume fs.CommaSepList will be like `--internetarchive-item-metadata="mediatype=web,collection=web,title=Rclone upload test"` but what if I want to use a comma in one of the metadata values? We actually do that for the description of the item.

Reply:
Using `[]string` works kind of by accident for command line options only. However it does not work for things set in the config file - you can only have one of those.

If we set this option only for use on the command line then we can have `[]string`. You do that with `Hide:     fs.OptionHideConfigurator`

Is that acceptable?